src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const currentState = state;
+
+  for (const action of actions) {
+    const { type, keysToRemove, extraData } = action;
+
+    switch (type) {
+      case 'clear':
+        for (const key in currentState) {
+          delete currentState[key];
+        };
+
+        break;
+
+      case 'addProperties':
+        Object.assign(currentState, extraData);
+
+        break;
+
+      case 'removeProperties':
+        for (const keyToRemove of keysToRemove) {
+          delete currentState[keyToRemove];
+        }
+
+        break;
+
+      default:
+        throw Error(`Wrong action: ${action.type}`);
+    }
+  }
+
+  return state;
```

Comment:
Returning 'state' is not necessary, as the function is supposed to modify the original object. Remove the 'return' statement.